vscode-dotty/package-lock.json

```diff
@@ -922,7 +912,7 @@
         },
         "readable-stream": {
           "version": "1.0.34",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```

Comment:
Can you remove all the changes that replace https by http ? My npm doesn't do that